Updating build index: /home/vsts/conda/conda-bld

TEST START: /tmp/tmp19iufolv/linux-64/sixgill-0.2.4-py36_1.tar.bz2
Adding in variants from internal_defaults
Reloading output folder: /home/vsts/conda/conda-bld
Mamba failed to solve:
 - sixgill 0.2.4.* py39_1
 - python
 - biopython
 - pysam >=0.9.0
 - python

with channels:
 - file:///tmp/tmp19iufolv

The reported errors are:
   Encountered problems while solving:
     - nothing provides requested sixgill 0.2.4.* py39_1
   

Leaving build/test directories:
  Work:
 /home/vsts/conda/conda-bld/work 
  Test:
 /home/vsts/conda/conda-bld/test_tmp 
Leaving build/test environments:
  Test:
source activate  /home/vsts/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh 
  Build:
source activate  /home/vsts/conda/conda-bld/_build_env 


